### PR TITLE
Thumbnail Filter

### PR DESCRIPTION
**Pull request by iolloyd**
[Review on Reviewable](https://reviewable.io/reviews/php-ffmpeg/php-ffmpeg/127)

### CODE REVIEW

**Comment by iolloyd** [comment]
Is the project still in active development? I would be interesting in helping out if need be.

File: src/FFMpeg/Filters/Video/ThumbnailFilter.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+ */
+
+namespace FFMpeg\Filters\Video;
+
+use FFMpeg\Filters\VideoFilterInterface;
+use FFMpeg\Format\VideoInterface;
+use FFMpeg\Media\Video;
+
+/**
+ * Class ThumbnailFilter
```
Comment (patkar) [inline src/FFMpeg/Filters/Video/ThumbnailFilter.php]: Obviously ^^

File: src/FFMpeg/Filters/Video/ThumbnailFilter.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+ *
+ */
+class ThumbnailFilter implements VideoFilterInterface
+{
+    /**
+     * @param     $videoFile
+     * @param     $duration
+     * @param int $count
```
Comment (patkar) [inline src/FFMpeg/Filters/Video/ThumbnailFilter.php]: Those are not method params..

**Comment by patkar** [comment]
@iolloyd May you add tests please?

**Comment by Romain** [comment]
@romainneutron Don't you think we should close this PR?

**Comment by jens1o** [comment]
any news on this?

**Comment by Romain** [comment]
I don't see the difference with the Frame filter.
I think we should just close this PR since it hasn't been updated for a long time.

**Comment by jens1o** [comment]
Okay, close it.

**Pull request closed without merging**